package.json

```diff
@@ -48,7 +48,7 @@
     "@ethersproject/address": "^5.0.8",
     "@ethersproject/bignumber": "^5.1.1",
     "@ethersproject/bytes": "^5.0.8",
-    "ethers": "^5.1.0"
+    "ethers": "^5.7.0"
```

Comment:
Can we check what would be the minimal version needed for beamer bridge, and add it as a `peerDependency`?

Reply:
Their [frontend repo](https://github.com/beamer-bridge/beamer/blob/main/frontend/package.json#L29) shows version 5.1.1.

What was the context of correlating it to the beamer bridge? Were they the ones having issues with versioning?